lib/rules/decorator-position.js

```diff
@@ -0,0 +1,157 @@
+'use strict';
+
+const ember = require('./ember');
+const utils = require('./utils');
+
+const defaultOptions = {
+  trackedOnSameLine: true,
+  computedWithArgumentsOnSameLine: false,
+  emberDataOnSameLine: true,
+
+  computedWithoutArgumentsOnSameLine: true,
+  actionOnSameLine: true,
+  dependentKeyCompatOnSameLine: false,
+};
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforces consistent decorator position on properties, methods, etc',
+      category: '',
```

Comment:
`Stylistic Issues` looks like the right category for this. Also remember to rebase on the latest master.

Reply:
thanks!